### PR TITLE
Some grammar changes

### PR DESCRIPTION
**Pull request by ericfode**

### CODE REVIEW

File: src/cats/protocols.cljc [hunk 1]
```diff
@@ -37,12 +37,12 @@
   "Abstraction that establishes a concrete type as a member of a context.
 
   A great example es Maybe monad type Just. It implements
```
Comment (yurrriq) [inline src/cats/protocols.cljc]: Maybe `s/es Maybe/is the Maybe/`.

File: src/cats/protocols.cljc [hunk 2]
```diff
@@ -73,8 +73,8 @@
     "Applies the function(s) inside af's context to the value(s)
      inside av's context while preserving the context.")
   (-pure [app v]
-    "Takes any context monadic value ctx and any value v, and puts
-     the value v in the most minimal context of same type of ctx"))
+    "Takes any context or monadic value (app) and any value (v), and puts
```
Comment (yurrriq) [inline src/cats/protocols.cljc]: I think I'd prefer backticks instead of parens for `app` and `v` here.

File: src/cats/protocols.cljc [hunk 3]
```diff
@@ -73,8 +73,8 @@
...
+     the value v in the most minimal context (normally mempty) of same type of app"))
```
Comment (yurrriq) [inline src/cats/protocols.cljc]: We should probably surround `v`, `mempty` and `app` in backticks here, too.

**Comment by yurrriq** [comment]
Thanks! I've made a few suggestions inline.

**Comment by ericfode** [comment]
Think i got them all.

**Comment by yurrriq** [comment]
Thanks, I'll review and merge later today.

**Comment by dialelo** [comment]
Thanks! :cake:

**Comment by yurrriq** [comment]
Thanks for merging, @dialelo.

**Pull request merged**